threewisemonkeys-as_robo_interact_s2_159.py

```
@@ -0,0 +1,117 @@
+        
+        # Serialize masks for JSON response
+        serialized_masks = serialize_masks(masks)
+        
+        return jsonify({
+            'status': 'success',
+            'num_masks': len(masks),
+            'masks': serialized_masks,
+            'generation_time': time.time() - mask_start
+        })
+        
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)